pkgs/games/gcs/default.nix

```diff
@@ -1,78 +1,65 @@
-{ lib, stdenv, fetchFromGitHub, runCommand
-, jdk8, ant
-, jre8, makeWrapper
+{ lib
+, buildGoModule
+, pkg-config
+, xorg
+, libGL
+, freetype
+, fontconfig
+, zlib
+, harfbuzz
+, openjpeg
+, jbig2dec
+, libjpeg_turbo
+, gumbo
+, mupdf
+, fetchFromGitHub
 }:
 
 let
+  ver = "5.15.0";
   gcs = fetchFromGitHub {
     owner = "richardwilkes";
     repo = "gcs";
-    rev = "gcs-4.8.0";
-    sha256 = "0k8am8vfwls5s2z4zj1p1aqy8gapn5vbr9zy66s5g048ch8ah1hm";
+    rev = "v${ver}";
+    sha256 = "sha256-bhvWTpUrlL3rTJm8Edaih5+j2dqjjG/aB4a2twXQI/4=";
   };
-  appleStubs = fetchFromGitHub {
-    owner = "richardwilkes";
-    repo = "apple_stubs";
-    rev = "gcs-4.3.0";
-    sha256 = "0m1qw30b19s04hj7nch1mbvv5s698g5dr1d1r7r07ykvk1yh7zsa";
-  };
-  toolkit = fetchFromGitHub {
-    owner = "richardwilkes";
-    repo = "toolkit";
-    rev = "gcs-4.8.0";
-    sha256 = "1ciwwh0wxk3pzsj6rbggsbg3l2f741qy7yx1ca4v7vflsma84f1n";
-  };
-  library = fetchFromGitHub {
-    owner = "richardwilkes";
-    repo = "gcs_library";
-    rev = "gcs-4.8.0";
-    sha256 = "085jpp9mpv5kw00zds9sywmfq31mrlbrgahnwcjkx0z9i22amz4g";
-  };
-in stdenv.mkDerivation rec {
+in buildGoModule rec {
   pname = "gcs";
-  version = "4.8.0";
-
-  src = runCommand "${pname}-${version}-src" { preferLocalBuild = true; } ''
-    mkdir -p $out
-    cd $out
+  version = ver;
 
-    cp -r ${gcs} gcs
-    cp -r ${appleStubs} apple_stubs
-    cp -r ${toolkit} toolkit
-    cp -r ${library} gcs_library
+  src = gcs;
+  nativeBuildInputs = [pkg-config];
+  buildInputs = [
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXrandr
+    xorg.libXinerama
+    xorg.libXi
+    xorg.libXxf86vm
```

Comment:
```suggestion
    libX11
    libXcursor
    libXrandr
    libXinerama
    libXi
    libXxf86vm
```

Reply:
> 
would this change work? wouldn't it be necessary to add a with xorg; to the top or is github not showing you adding a with xorg; line

Reply:
Include the corresponding lines in the input attrset.